docs/base-api.md

```diff
@@ -481,6 +316,33 @@ $baseApi->listDrmCertificates(
     ```
     A support ticket has been created at bunny.net regarding this issue.
 
+### Integrations
```

Comment:
This is necessary to have in order to use the "Add Edge Script" endpoint when creating/deploying through GitHub, as it requires an `IntegrationId`, which can be retrieved from the `/integration/github` endpoint.